daemon/stop.go

```diff
@@ -78,12 +78,16 @@ func (daemon *Daemon) containerStop(container *containerpkg.Container, seconds i
 		logrus.Infof("Container %v failed to exit within %d seconds of signal %d - using the force", container.ID, seconds, stopSignal)
 		// 3. If it doesn't, then send SIGKILL
 		if err := daemon.Kill(container); err != nil {
-			// Wait without a timeout, ignore result.
-			<-container.Wait(context.Background(), containerpkg.WaitConditionNotRunning)
-			logrus.Warn(err) // Don't return error because we only care that container is stopped, not what function stopped it
+			logrus.Errorf("Container %v failed to SIGKILL container: %s", container.ID, err)
```

Comment:
`logrus.WithError(err).WithField("container", container.ID).Error("Error killing the container")`